src/couch_dist/README.md

```diff
@@ -0,0 +1,160 @@
+# couch_dist
+
+`couch_dist` implements a custom distribution protocol -- `couch`, which allows
+nodes to connect using different protocols, such as Transport Layer Security
+(TLS) and Transmission Control Protocol (TCP).
```

Comment:
Erlang communicates with its own protocol over TCP. It can also be configured to run its protocol over a TLS connection which itself is over TCP.